packages/commonwealth/server/migrations/20230509184335-remove-chain-categories.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(async (transaction) => {
+      try {
+        await queryInterface.addColumn(
+          'Chains',
+          'category',
+          {
+            type: Sequelize.STRING,
```

Comment:
Can we use a JSON or JSONB format rather than string? Or even an array of strings.

Reply:
Converted to JSONB. I think this will give us the most flexibility in the future for any change/additions to the data structure